src/ssiApi/apiAuth/apiAuth.ts

```diff
@@ -0,0 +1,29 @@
+import fetch from "node-fetch"
+import { IgenerateToken } from "./IAuth";
+export class ApiAuth {
+    private apiKey: string;
+    constructor(apiKey) {
+        if (!apiKey || apiKey.trim() === "") {
+            throw new Error("HID-SSI_SDK:: Error: Please Provide apiKey")
+        }
+        this.apiKey = apiKey
+    }
+    async generateAccessToken(): Promise<IgenerateToken> {
+        const studioApiUrl = "https://api.entity.hypersign.id/api/v1/app/oauth"
```

Comment:
This should not be hardcoded here, put it in config

Reply:
ya i have added it in separate file now. Yet not commited